### PR TITLE
chrony: enable virtual PHC in chrony.conf

### PR DESCRIPTION
**Pull request by cjp256**
For more information, check out:
https://docs.microsoft.com/en-us/azure/virtual-machines/linux/time-sync

Signed-off-by: Chris Patterson <cpatterson@microsoft.com>

###### Test Methodology
Built, installed, validated chrony sources with `chronyc sources`.

### CODE REVIEW

**Comment by cjp256** [comment]
As this change should only affect Azure images, will move this logic into https://github.com/microsoft/CBL-Mariner/blob/2.0/toolkit/imageconfigs/additionalconfigs/configure-image.sh

File: SPECS/chrony/chrony.spec [hunk 1]
```diff
@@ -79,11 +79,15 @@ EOF
...
 # - enable leapsectz to get TAI-UTC offset and leap seconds from tzdata
 # - enable keyfile
+# - enable virtual PHC
 sed -e 's|^pool.*|server time.windows.com|' \
     -e 's|#\(leapsectz\)|\1|' \
     -e 's|#\(keyfile\)|\1|' \
         < examples/chrony.conf.example2 > chrony.conf
 
+echo -e "\n# Enable virtual PTP clock source." >> chrony.conf
+echo -e "refclock PHC /dev/ptp_hyperv poll 3 dpoll -2 offset 0" >> chrony.conf
```
Comment (hbeberman) [inline SPECS/chrony/chrony.spec:89]: I'm concerned about the implications of including this by default in every chrony.conf.
I agree with setting this in Azure images but it'll impact our edge baremetal and VM configs if it's here.

Comment (cjp256) [reply SPECS/chrony/chrony.spec:89]: Will fix.

Comment (cjp256) [reply SPECS/chrony/chrony.spec]: Done, though I can't reasonably test this.

**Review by anphel31 — APPROVED** [review]

**Review by christopherco — CHANGES_REQUESTED** [review]
We're applying this ptp change as part of a bigger chrony conf update - https://github.com/BettyRain/CBL-Mariner/commit/f1c2e1dcfe9e55bfa1fbcc5fa844d59edca95fb1  We're doing final testing on these changes and will open a PR shortly

**Comment by christopherco** [comment]
Full chrony.conf PR is up: https://github.com/microsoft/CBL-Mariner/pull/3603

**Comment by christopherco** [comment]
Closing this PR. Functionality of this PR has been merged in with https://github.com/microsoft/CBL-Mariner/pull/3603

**Comment by cjp256** [comment]
> Closing this PR. Functionality of this PR has been merged in with #3603

Thanks! Sorry it fell off my plate!

**Pull request closed without merging**